neurons/miners/miner.py

```diff
@@ -194,6 +223,9 @@ async def query_blacklist(self, synapse: protocol.Query) -> typing.Tuple[bool, s
     async def challenge_blacklist(self, synapse: protocol.Challenge) -> typing.Tuple[bool, str]:
         return blacklist.base_blacklist(self, synapse=synapse)
 
+    async def llm_query_blacklist(self, synapse: protocol.LlmQuery) -> typing.Tuple[bool, str]:
+        return blacklist.base_blacklist(self, synapse=synapse)
```

Comment:
we should enable this synapse for validators giving emissions only, please chat with @wendellswa06 or @0xMarx about it

Reply:
got it

Reply:
It will be done as part of this task:
https://linear.app/sn15/issue/SN1-369/blacklisting-all-validators-not-giving-emissions-for-query-endpoints